asset-transfer-basic/rest-api-typescript/src/config.ts

```diff
@@ -142,8 +153,8 @@ export const chaincodeName = env
  */
 export const commitTimeout = env
   .get('HLF_COMMIT_TIMEOUT')
-  .default('3000')
-  .example('3000')
+  .default('300')
+  .example('300')
```

Comment:
"magic value" => how do I choose what value should go here?

Reply:
Pretty sure it is magic
If I find any useful documentation, I'll include a link in a follow on PR